swagger_server/controllers/survey_response.py

```diff
@@ -127,6 +137,39 @@ def _get_collection_exercise(self, collection_exercise_id):
             ons_env.logger.debug('Collection Exercise not found')
         return collection_exercise
 
+
+    def _get_survey_service(self, survey_id):
+        """
+        Used to get the SurveyRef number from the survey service. This is used to construct the file name of an
```

Comment:
The  comment should really be what the function is doing, not what the thing calling it will be using it for. So the comment should be get survey data from the survey service etc